authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerAuthorizationHDFSPluginIT.java

```diff
@@ -150,7 +150,7 @@ public void testTranslateMetadataObject() {
                         (PathBasedMetadataObject) securableObject;
                     Assertions.assertEquals(fileset.fullName(), pathBasedMetadataObject.fullName());
                     Assertions.assertEquals(
-                        PathBasedMetadataObject.FILESET_PATH, securableObject.type());
+                        PathBasedMetadataObject.FILESET_PATH, pathBasedMetadataObject.type());
```

Comment:
Did you mean to change this?

Reply:
Sorry for the confusion. This change was not intended. I will revert it.